python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -59,67 +52,87 @@ def core_info(engine):
     """
     import sgtk
     from sgtk.commands.core_upgrade import TankCoreUpdater
-    logger = sgtk.LogManager.get_logger(LOGGER_NAME)
-
-    code_css_block = "display: block; padding: 0.5em 1em; border: 1px solid #bebab0; background: #faf8f0;"
 
     # Create an upgrader instance that we can query if the install is up to date.
     installer = TankCoreUpdater(
         engine.sgtk.pipeline_configuration.get_install_location(),
-        logger,
+        engine._log,
     )
 
     cv = installer.get_current_version_number()
     lv = installer.get_update_version_number()
 
-    logger.info(
+    engine.log_info(
         "You are currently running version %s of the Shotgun Pipeline Toolkit." % cv
     )
 
     if not engine.sgtk.pipeline_configuration.is_localized():
-        logger.info("")
-        logger.info(
-            "Your core API is located in <code>%s</code> and is shared with other "
+        engine.log_info(
```

Comment:
I added comments before the first log_info call for each of the two functions here that make use of markdown.